urllib3/connectionpool.py

```diff
@@ -192,6 +194,8 @@ def __init__(self, host, port=None, strict=False,
         for _ in xrange(maxsize):
             self.pool.put(None)
 
+        self.transport_security_manager = transport_security_manager or TransportSecurityManager()
```

Comment:
So I still think this is being initialized in the wrong place. If we're going to initialize it automatically at all, it should be at the PoolManager level or higher.